pkg/featureflag/featureflag.go

```diff
@@ -25,8 +25,10 @@ import (
 // They are activated via environment variables, starting with "RATIFY_", ex: RATIFY_DYNAMIC_PLUGINS=1
 // Remember to capture changes in the usage guide and release notes.
 var (
-	DynamicPlugins = newFeatureFlag("DYNAMIC_PLUGINS", false)
-	CertRotation   = newFeatureFlag("CERT_ROTATION", false)
+	DynamicPlugins  = newFeatureFlag("DYNAMIC_PLUGINS", false)
+	UseRegoPolicy   = newFeatureFlag("USE_REGO_POLICY", false)
```

Comment:
wondering whats our road map for GA, will be slowly switch for USE_REGO as the default?